src/vm/docs/mod.rs

```diff
@@ -737,6 +747,114 @@ definition (i.e., you cannot put a define statement in the middle of a function
 "
 };
 
+const MINT_TOKEN: SpecialAPI = SpecialAPI {
+    name: "mint-token!",
+    input_type: "TokenName, int, principal",
+    output_type: "(response bool int)",
+    signature: "(mint-token! token-name amount recipient)",
+    description: "`mint-token!` is used to increase the token balance for the `recipient` principal for a token
+type defined using `define-token`. The increased token balance is _not_ transfered from another principal, but
+rather minted.",
+    example: "
+(define-token stackaroo)
+(mint-token! stackaroo 100 tx-sender)
+"
+};
+
+const MINT_ASSET: SpecialAPI = SpecialAPI {
+    name: "mint-asset!",
+    input_type: "AssetName, A, principal",
+    output_type: "(response bool int)",
+    signature: "(mint-asset! asset-class asset-identifier recipient)",
+    description: "`mint-asset!` is used to instantiate an asset and set that asset's owner to the `recipient` principal.
+The asset must have been defined using `define-asset`, and the supplied `asset-identifier` must be of the same type specified in
+that definition.
+
+If an asset identified by `asset-identifier` _already exists_, this function will return an error with the following error code:
+
+`(err 1)`
+",
+    example: "
+(define-asset stackaroo (buff 40))
+(mint-token! stackaroo \"Roo\" tx-sender)
+"
+};
+
+const GET_OWNER: SpecialAPI = SpecialAPI {
+    name: "get-owner",
+    input_type: "AssetName, A",
+    output_type: "(optional principal)",
+    signature: "(get-owner asset-class asset-identifier)",
+    description: "`get-owner` returns the owner of an asset, identified by `asset-identifier`, or `none` if the asset does not exist.
+The asset type must have been defined using `define-asset`, and the supplied `asset-identifier` must be of the same type specified in
+that definition.",
+    example: "
+(define-asset stackaroo (buff 40))
+(get-owner stackaroo \"Roo\")
+"
+};
+
+
+const GET_BALANCE: SpecialAPI = SpecialAPI {
+    name: "get-balance",
+    input_type: "TokenName, principal",
+    output_type: "int",
+    signature: "(get-balance token-name principal)",
+    description: "`get-balance` returns `token-name` balance of the principal `principal`.
+The token type must have been defined using `define-token`.",
+    example: "
+(define-token stackaroos)
+(get-balance stackaroos tx-sender)
+"
+};
+
+const TOKEN_TRANSFER: SpecialAPI = SpecialAPI {
+    name: "transfer-token!",
+    input_type: "TokenName, int, principal, principal",
+    output_type: "(response bool int)",
+    signature: "(transfer-token! token-name amount sender recipient)",
+    description: "`transfer-token!` is used to increase the token balance for the `recipient` principal for a token
+type defined using `define-token` by debiting the `sender` principal.
+
+This function returns (ok true) if the transfer is successful. In the event of an unsuccessful transfer it returns
+one of the following error codes:
```

Comment:
Will it ever return `(ok false)`?

Reply:
No, we don't have Void types, which is why a number of functions return true in all cases.